nixos/modules/misc/ids.nix

```diff
@@ -666,6 +667,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      goeland = 327;
```

Comment:
We no longer want to add new IDs here unless absolutely necessary. Can we just use users.users?

Reply:
Oh okay, removed 👍🏽